Google.Cloud.EntityFrameworkCore.Spanner.IntegrationTests/SpannerSampleFixture.cs

```diff
@@ -128,6 +128,13 @@ private void CreateTables()
             var statements = script.Split(";");
             for (var i = 0; i < statements.Length; i++)
             {
+                // Remove license header from script
```

Comment:
Is it at all necessary to remove it from the script? The copy right header is also a valid comment in Cloud Spanner SQL syntax, so I would expect Cloud Spanner to just ignore the comment and execute the actual statement that follows it.

Reply:
I was also thinking same but actually it's throwing Invalid Argument Exception:
`
Status(StatusCode="InvalidArgument", Detail="Error parsing Spanner DDL statement: /* Copyright 2021 Google LLC\r\n* \r\n* Licensed under the Apache License, Version 2.0 (the \"License\")\r\n* you may not use this file except in compliance with the License.\r\n* You may obtain a copy of the License at\r\n* \r\n*     https://www.apache.org/licenses/LICENSE-2.0\r\n* \r\n* Unless required by applicable law or agreed to in writing, software\r\n* distributed under the License is distributed on an \"AS IS\" BASIS,\r\n* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.\r\n* See the License for the specific language governing permissions and\r\n* limitations under the License.\r\n*/\r\n\r\nCREATE TABLE Singers (\r\n  SingerId  INT64 NOT NULL,\r\n  FirstName STRING(200),\r\n  LastName  STRING(200) NOT NULL,\r\n  FullName  STRING(400) NOT NULL AS (COALESCE(FirstName || \' \', \'\') || LastName) STORED,\r\n  BirthDate DATE,\r\n  Picture   BYTES(MAX),\r\n) PRIMARY KEY (SingerId) : Syntax error on line 1, column 1: Encountered \'/\' while pa...", DebugException="Grpc.Core.Internal.CoreErrorDetailException: {"created":"@1620917204.223000000","description":"Error received from peer ipv4:142.250.183.234:443","file":"T:\src\github\grpc\workspace_csharp_ext_windows_x64\src\core\lib\surface\call.cc","file_line":1062,"grpc_message":"Error parsing Spanner DDL statement: /* Copyright 2021 Google LLC\r\n* \r\n* Licensed under the Apache License, Version 2.0 (the \"License\")\r\n* you may not use this file except in compliance with the License.\r\n* You may obtain a copy of the License at\r\n* \r\n*     https://www.apache.org/licenses/LICENSE-2.0\r\n* \r\n* Unless required by applicable law or agreed to in writing, software\r\n* distributed under the License is distributed on an \"AS IS\" BASIS,\r\n* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.\r\n* See the License for the specific language governing permissions and\r\n* limitations under the License.\r\n*/\r\n\r\nCREATE TABLE Singers (\r\n  SingerId  INT64 NOT NULL,\r\n  FirstName STRING(200),\r\n  LastName  STRING(200) NOT NULL,\r\n  FullName  STRING(400) NOT NULL AS (COALESCE(FirstName || \' \', \'\') || LastName) STORED,\r\n  BirthDate DATE,\r\n  Picture   BYTES(MAX),\r\n) PRIMARY KEY (SingerId) : Syntax error on line 1, column 1: Encountered \'/\' while pa...","grpc_status":3}")
`

Reply:
Strange, it seems that Cloud Spanner only accepts comments in SQL queries and DML statements, but not in DDL statements.

Reply:
Ah, that's a shame. Ok we can just remove those and I'll explain to the approvers that it's not possible to add headers to SQL files. I also noticed that the sql files in the JDBC repo don't have a header either.

Reply:
I take my last comment back. In the JDBC repo we have https://github.com/googleapis/java-spanner-jdbc/blob/master/src/test/resources/com/google/cloud/spanner/jdbc/ITSqlScriptTest_CreateTables.sql, which has DDL statements so Spanner should be able to accept this, no?

Reply:
I had forgotten as well, but the JDBC driver (and the Java client library) strip the copyright header before executing any statements from these SQL scripts: https://github.com/googleapis/java-spanner/blob/eb25eae4a2f83fd8c35907a87785baf042889233/google-cloud-spanner/src/test/java/com/google/cloud/spanner/connection/AbstractSqlScriptVerifier.java#L169